torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -1406,8 +1407,9 @@ class AliasDb::WorkingSet {
     if (mover_ && users.count(mover_)) {
       return true;
     }
-    return std::any_of(nodes_.begin(), nodes_.end(), [&](Node* node) {
-      return users.count(node) != 0;
+
+    return std::any_of(users.begin(), users.end(), [&](Node* node) {
```

Comment:
Nit: rename argument in lambda to `user` to reflect that it is actually part of `users`

Reply:
done